### PR TITLE
ci: gotest container spec for unit test

### PR DESCRIPTION
**Pull request by mikkosest**
Container for unit test

### CODE REVIEW

File: resources/docker/gotest/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM golang:1.11-alpine3.7
+
+RUN apk add --update \
+    git \
+    glide \
+    make \
+    sudo \
```
Comment (maelk) [inline resources/docker/gotest/Dockerfile]: why sudo ? you have not changed the user (root) so you should not have permission issues. Which script is using it ?

File: resources/docker/gotest/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		/usr/local/go/pkg/obj \
+	; \
+	apk del .build-deps; \
+	\
+	export PATH="/usr/local/go/bin:$PATH"; \
+	go version
+
+ENV GOPATH /go
+
+RUN apk add --update \
```
Comment (maelk) [inline resources/docker/gotest/Dockerfile]: why separate the apk steps ?

File: resources/docker/gotest/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    mv /go/bin/dep /usr/local/bin && \
+    curl -LO https://raw.githubusercontent.com/Nordix/cluster-api-provider-baremetal/master/tools/install_kubebuilder.sh && \
+    chmod +x ./install_kubebuilder.sh \
+    && ./install_kubebuilder.sh \
+    && rm ./install_kubebuilder.sh && \ 
+    mv /kubebuilder /usr/local && \
+    cp /usr/local/kubebuilder/bin/kubebuilder /usr/local/bin
+
+COPY run-test.sh /usr/bin/testentrypoint
+ENTRYPOINT ["/usr/bin/testentrypoint"]
```
Comment (maelk) [inline resources/docker/gotest/Dockerfile]: maybe CMD rather than ENTRYPOINT ?

Comment (mikkosest) [reply resources/docker/gotest/Dockerfile]: Done

**Pull request merged**